src/backend/distributed/commands/create_distributed_table.c

```diff
@@ -71,6 +74,12 @@
 /* Replication model to use when creating distributed tables */
 int ReplicationModel = REPLICATION_MODEL_COORDINATOR;
 
+typedef enum
```

Comment:
Adding a comment about the differences of values would help the readers. What do they represent?